package.json

```diff
@@ -10,9 +10,10 @@
   ],
   "license": "SEE LICENSE IN LICENSE.md",
   "scripts": {
-    "build": "tsc -p ./tsconfig.json",
+    "build": "rm -rf dist && tsc -p ./tsconfig.json",
```

Comment:
maybe invoke clean directly instead of duplicating

Reply:
I modified it, though `yarn` vs `npm` run will now be something to watch out for. Not like we're switching any time soon tho.